learn/pkg/learn/question.go

```diff
@@ -455,6 +543,23 @@ func (m *QuestionModel) buildAnswerChoicesField(block markdown.Block) error {
 	return nil
 }
 
+func (m *QuestionModel) buildAnswerTextField(block markdown.Block) error {
+	if m.AnswerText != nil || m.Question == nil {
+		return nil
+	}
+	var err error
+	m.AnswerText, err = NewRenderer(block, answerField, m.Filename())
```

Comment:
it seems odd that NewRenderer can return nil without an error - a little unexpected perhaps

Reply:
ok, after some debate we settled on renaming it to `AsRenderer`